sdk/servicebus/service-bus/README.md

```diff
@@ -66,7 +66,17 @@ You can instantiate this class using its constructors:
 
 ### Key concepts
 
-Once you have initialized the [ServiceBusClient][sbclient] class you will be able to:
+Once you've initialized a `ServiceBusClient`, you can interact with the primary resource types within a
+Service Bus Namespace, of which multiple can exist and on which actual message transmission takes place,
+the namespace often serving as an application container:
```

Comment:
That is one very long complicated sentence!
Are we copying this over from somewhere else?

Reply:
Can we simplify this with

> A Service Bus namespace contains the below resources on which message transmission can take place:

Reply:
In the meeting everyone was pretty happy with Python's so that's where I snagged it from.

Reply:
CC: @KieranBrantnerMagee, @yunhaoling - in case they're interested in our edits.

Reply:
Updated.